mat-chalmers.cabal

```diff
@@ -14,6 +14,7 @@ data-files:
 library
   default-language: Haskell2010
   hs-source-dirs: src
+  ghc-options: -Wall -fwarn-unused-binds -fwarn-unused-imports
```

Comment:
I think modern ghc would rather you -Wunused-binds and -Wunused-imports, though both work.

Reply:
So no need to change anything, just sharing knowledge :)

Reply:
Thank you! I'm adding this to the next PR, it would be a shame to let knowledge go to waste. :)

Reply:
https://downloads.haskell.org/~ghc/8.6.3/docs/html/users_guide/using-warnings.html#options-sanity